setup.py

```diff
@@ -7,12 +7,14 @@
 from setuptools import find_packages, setup
 
 
-def get_install_required():
-    with open("./requirements.txt", "r") as reqs:
+def get_install_required(extra=None):
+    extra = f"-{extra}" if extra is not None else ""
+    with open(f"./requirements{extra}.txt", "r") as reqs:
         requirements = reqs.readlines()
     return [r.rstrip() for r in requirements]
 
 
+
```

Comment:
this doesn't satisfy the `pre-commit` hooks, make sure you run `pre-commit install`